docs/scan-sequence.puml

```diff
@@ -0,0 +1,34 @@
+@startuml
+autonumber
+
+actor       User       as user
+participant "Workload controller" as wc
+entity      Pod      as pod
+participant "CIS controller" as cc
+entity      CIS      as cis
+participant "Scan Job controller" as jc
+entity      Job      as job
+
+user -> pod : create
+activate pod 
+
+wc <<- pod : watch
+activate wc
+wc -> cis : create
+deactivate wc
```

Comment:
```suggestion
deactivate wc
activate cis
```

Example:

![](http://www.plantuml.com/plantuml/png/RO_1IiGm48RlynHXx_i2iiY29pq9YtXE_mHiRPEIJ0n27zwCiOQKpIapVF_7FzV8Ed1QPjA9lKlBO0EH1lkWwdkBTl_gg5APQ2sH4UEg7QlJkm-VizT6mJiEVfvjE0cPGTRnoDyr--hDlsJrfdSyFT-ESVHn8VOunjZ7Rz1ElVZXw9XwXn2xO_83aHoZpe_IHpqe1AlPoktZL_dyReaorELo_aEoPdmq8eCoYa0QDO6n3K23fJMXsYf_i044iKctkpOqM-s_RHlHL9LbiHkDxR0seC2jRbgDR2DhJh4BWUXgdKdB_0C0)

Reply:
Nice, it shows the lifecycle of the CIS better 👍

Reply:
Thanks @mikaelol!